src/nni_manager/training_service/pai/paiTrainingService.ts

```diff
@@ -330,10 +206,10 @@ class PAITrainingService implements TrainingService {
             uri: `http://${this.paiClusterConfig.host}/rest-server/api/v1/user/${this.paiClusterConfig.userName}/jobs/${trialJobDetail.paiJobName}/executionType`,
             method: 'PUT',
             json: true,
-            body: {'value' : 'STOP'},
+            body: {value: 'STOP'},
             headers: {
-                "Content-Type": "application/json",
-                "Authorization": 'Bearer ' + this.paiToken
+                'Content-Type': 'application/json',
+                Authorization: `Bearer ${this.paiToken}`
```

Comment:
one uses "'" and the other does not?

Reply:
This is updated per tslint hint, because there is '-' in Content-Type, must be quoted.

Reply:
got it.